Notification.Application/Commands/Broker/UnsubscriptionCommand.cs

```diff
@@ -0,0 +1,48 @@
+﻿using MediatR;
+using Notification.Application.Handler;
+using Notification.Application.Logger;
+using Notification.Contracts.DataTransferObject.Broker;
+using Notification.Data.Repositories.Interface;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Notification.Application.Commands.Broker
+{
+    public class UnsubscriptionCommand : IRequest
+    {
+        public UnsubscriptionCommand(Unsubscription unsubscription)
```

Comment:
Got it, 
In my previous comment name command as SubscribeCommand. 
Here: UnsibscribeCommand.